packages/terra-switch/src/terra-dev-site/doc/example/DefaultSwitch.jsx

```diff
@@ -0,0 +1,15 @@
+import React, { useState } from 'react';
+import Switch from 'terra-switch';
+
+export default () => {
+  const [value, setValue] = useState(true);
+  return (
+    <div>
+      <Switch
+        isOn={value}
```

Comment:
we could Keep Default switch by passing value as off and add one more example For 'Switch On'.

Reply:
Added with this [commit](82255613e7490a31855ffcb6114fce680a8934c6)